tests/test_integrated_channels/test_moodle/test_client.py

```diff
@@ -80,6 +101,47 @@ def test_get_course_id(self):
         )
         assert client.get_course_id('course:test_course') == 2
 
+    def test_successful_create_content_metadata(self):
+        """
+        Test core logic of create_content_metadata to ensure
+        query string we send to Moodle is formatted correctly.
+        """
+        expected_data = SERIALIZED_DATA.copy()
+        # expected_data['courses[0][id]'] = self.moodle_course_id
+        expected_data['wsfunction'] = 'core_course_create_courses'
+
+        client = MoodleAPIClient(self.enterprise_config)
+        client._post = unittest.mock.MagicMock(name='_post', return_value=SUCCESSFUL_RESPONSE)  # pylint: disable=protected-access
```

Comment:
just a thought- is there an easy way we can disable this pylint warning for all test files?